index.html

```diff
@@ -9,11 +9,87 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+
+<header>
+    
+    
+    <nav id="navbar" class="nav">
+        <img id="logo" src="img/karma-logo.svg" alt=# width="30" height="50" />  
```

Comment:
```suggestion
        <img id="logo" src="img/karma-logo.svg" alt="Karma logo" width="30" height="50" />  
```
All images should have an `alt` description which describes the image. Same applies to the other images in this file